api/hooks/api-session.hook.ts

```diff
@@ -2,14 +2,13 @@ import { useAuthContext } from '../contexts/auth.context';
 import { useAuth } from './auth.hook';
 
 export interface ApiSessionInterface {
-  isLoggedIn: boolean;
   getSignMessage: (address: string) => Promise<string>;
```

Comment:
because it was only funnelled through of `AuthContext`, therefore I have thought it won't make sense and deleted it. It is a point on my list, we need to discuss. Because I think the exchange setup is not perfect yet and can still be improved. That is one of the improvements